packages/mdc-slider/foundation.ts

```diff
@@ -27,13 +27,13 @@ import {EventType, SpecificEventListener} from '@material/base/types';
 import {MDCSliderAdapter} from './adapter';
 import {cssClasses, numbers, strings} from './constants';
 
-type UpEventType = 'mouseup' | 'pointerup' | 'touchend';
-type DownEventType = 'mousedown' | 'pointerdown' | 'touchstart';
-type MoveEventType = 'mousemove' | 'pointermove' | 'touchmove';
-type MouseLikeEvent = MouseEvent | PointerEvent | TouchEvent;
+type UpEventType = 'mouseup'|'pointerup'|'touchend';
```

Comment:
Can we revert changes to the component?

Reply:
Keeping as is, since these are style fixes from internal `g4 fix`.